src/mysky/index.ts

```diff
@@ -555,6 +544,21 @@ export class MySky {
     return { data: entry.data };
   }
 
+  /**
+   * Deletes the entry data at the given path, if the user has given Discoverable
+   * Write permissions.
+   *
+   * @param path - The data path.
+   * @param [customOptions] - Additional settings that can optionally be set.
+   * @returns - An empty promise.
+   * @throws - Will throw if the user does not have Discoverable Write permission on the path.
+   */
+  async deleteEntryData(path: string, customOptions?: CustomSetJSONOptions): Promise<void> {
+    // Validation is done in `setEntryData`.
```

Comment:
The function is bound to the client and exposed, right? 
I think that means validation is required as it can be called separately.

Reply:
This function is actually not being used anywhere. The only `deleteEntryData` that's being used is the one from `skydb`.